build/parse.y

```diff
@@ -164,7 +164,8 @@ package build
 %left  _AND
 %left  '<' '>' _EQ _NE _LE _GE _NOT _IN
 %left  '+' '-'
-%left  '*' '/' '%'
+%left  '*' '/' '%',
```

Comment:
what is the trailing comma?

Reply:
A typo, although it doesn't affect the behavior (i.e. the generated parse.y.go is not affected). Removed.